planning/costmap_generator/CMakeLists.txt

```diff
@@ -33,6 +33,13 @@ target_link_libraries(costmap_generator_lib
   FLANN::FLANN
 )
 
+if(${PCL_VERSION} GREATER_EQUAL 1.12.1)
+  find_package(Qhull REQUIRED)
+  target_link_libraries(costmap_generator_lib
+    QHULL::QHULL
+  )
+endif()
```

Comment:
Could you link some information related to this issue?

Reply:
@kenji-miyake I updated the description.